### PR TITLE
Add hydra-roles for admin user

### LINKED ISSUE #878: Repository manager
Does Admin Sets have any bearing on this?

### PR DESCRIPTION
**Pull request by crowesn**
Resolves #878

### CODE REVIEW

**Comment by crowesn** [comment]
I'll work on a unit test for this.

**Comment by crowesn** [comment]
Depends on #908

**Comment by hortongn** [comment]
@crowesn you commented a few days ago on #878 asking if we should hold on this.  Do we need to discuss or is it safe to review/merge this PR?  (I'm looking for PRs to review and everything but this one is either assigned, mine, or failing.)

**Comment by crowesn** [comment]
I think we agreed that this can be merged.

File: spec/models/sufia/ability_spec.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  describe "a user in the admin group" do
+    let(:user) { create(:user) }
+    before { allow(user).to receive_messages(groups: ['admin', 'registered']) }
+    it { is_expected.to be_able_to(:create, FileSet) }
+    it { is_expected.to be_able_to(:create, TinymceAsset) }
+    it { is_expected.to be_able_to(:create, ContentBlock) }
+    it { is_expected.to be_able_to(:update, ContentBlock) }
+    it { is_expected.to be_able_to(:read, ContentBlock) }
+    it { is_expected.to be_able_to(:read, Sufia::Statistics) }
+    it { is_expected.to be_able_to(:read, :admin_dashboard) }
```
Comment (scherztc) [inline spec/models/sufia/ability_spec.rb]: I really like this test.

**Pull request merged**